sdk/core/core-http/src/fetchHttpClient.ts

```diff
@@ -18,14 +18,23 @@ interface FetchError extends Error {
   type?: string;
 }
 
+/**
+ * String URLs.
+ */
 export type CommonRequestInfo = string; // We only ever call fetch() on string urls.
 
+/**
+ * An object containing information about the outgoing HTTP request.
```

Comment:
Why does it have “Init” in its name? I wonder if that’s important. I should continue to dig.